dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/FunctionsViewExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Azure.AI.OpenAI;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
+
+internal static class FunctionsViewExtensions
+{
+    public static IList<FunctionDefinition> ToFunctionDefinitions(this FunctionsView functionsView)
+    {
+        var functionDefinitions = new List<FunctionDefinition>();
+
+        ConcurrentDictionary<string, List<FunctionView>> nativeFunctions = functionsView.NativeFunctions;
```

Comment:
nit: no need to declare the variables if you can use `functionsView.NativeFunctions` and `functionsView.SemanticFunctions` directly in the foreach loops below.